From 736ae3de4072faa38f361e86862b85f7ffea23f7 Mon Sep 17 00:00:00 2001
From: Jia Zhang <lans.zhang2008@gmail.com>
Date: Tue, 22 Aug 2017 17:32:47 +0800
Subject: [PATCH] Fix the world build failure due to the missing rule of
 generating shim.key

Upstream-Status: Pending

shim.key is not given without feature/mok-secure-boot, the script
make-certs already integrated in shim is able to generate it and shim.crt
for signing. However, the commit 79c0d3ab3964ff03483277a515aaf50016bbe786
forgets to add the rule of generating shim.key, causing the world build
failure.

Signed-off-by: Jia Zhang <lans.zhang2008@gmail.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 1dd9fbc8..48482628 100644
--- a/Makefile
+++ b/Makefile
@@ -54,7 +54,7 @@ FALLBACK_SRCS = $(foreach source,$(ORIG_FALLBACK_SRCS),$(TOPDIR)/$(source))
 
 all: $(TARGETS)
 
-shim.crt:
+shim.crt shim.key:
 	$(TOPDIR)/make-certs shim shim@xn--u4h.net all codesign 1.3.6.1.4.1.311.10.3.1 </dev/null
 
 shim.cer: shim.crt
-- 
2.25.1

